test-clients/src/main/resource/contract/solidity/hip-206/IHederaTokenService.sol

```diff
@@ -758,4 +758,8 @@ interface IHederaTokenService {
     function getTokenType(address token)
         external returns
         (int64 responseCode, int32 tokenType);
+
+    /// Initiates a Redirect For Token
+    /// @param data The function selector from the ERC20 interface + the bytes input for the function called
```

Comment:
We can add a javadoc description also for the `token` parameter.

Reply:
Missed that one, added